packages/block-editor/src/components/block-list/index.js

```diff
@@ -135,7 +183,7 @@ export default function BlockList( settings ) {
 	);
 }
 
-BlockList.__unstableElementContext = elementContext;
+BlockList.useRootPortal = useRootPortal;
```

Comment:
Would be good to also make this unstable for now, but eventually this should be exported in the block editor package as an official API.

Reply:
I guess we can start private.